### ISSUE TITLE
Weird mode-line colors

### ISSUE THREAD

**Issue by maikol-solis** [rougier/nano-theme#51]
Hi! 

Today I tried nano-light since a long ago and I have these weird colors in the mode-line 

![image](https://github.com/rougier/nano-theme/assets/4078489/afa7e8ec-d175-4ea4-8461-91c9500c15db)

The colors for mode-line 
![image](https://github.com/rougier/nano-theme/assets/4078489/d9c34d65-3568-4158-a4c4-efface34a3c6)

And for mode-line-inactive 
![image](https://github.com/rougier/nano-theme/assets/4078489/e8671397-8d01-4a1f-b883-638ce4cbddc2)

In the code everything looks right, but I don't know if I missed something 

https://github.com/rougier/nano-theme/blob/9d64dc167883835a6dc3a6d286c98dbbf7e95a96/nano-theme-support.el#L692-L696

Thanks.

**Comment by rougier** [comment]
I think the theme is missing the mode-line-active that is a new face (I think). Can you check what's the face for mode-line-active (if it exists on your version) ?

**Comment by maikol-solis** [comment]
Yes, mode-line-active inherits the face from mode-line. 

![image](https://github.com/rougier/nano-theme/assets/4078489/2f45fc00-0fee-49e5-8c8a-8b5806ea59c5)

**Comment by maikol-solis** [comment]
Using   `custom-set-faces` I added these two configs 

```elisp
 `(mode-line :inherit nano-subtle
    :background nil
    :foreground nil
    :box (:color ,nano-light-faded :style nil))
  `(mode-line-inactive
    :inherit nano-subtle-i
    :foreground nil
    :background nil
    :box nil)
```

An now the active mode-line looks like this 

![image](https://github.com/rougier/nano-theme/assets/4078489/771d90b0-56ed-471a-b7be-22a9a10813e7)

And the inactive one like this 

![image](https://github.com/rougier/nano-theme/assets/4078489/e3103736-5207-439f-8c57-2c1337339f67)


It should be abstracted into the nano-theme structure, but I, personally, think it looks better. 

WDYT?

**Comment by rougier** [comment]
I looks nice! ActuallyI've tested the box in active mode before removing it because I found it a bit ditractng. A for the inactive mode, it's a bit too washed out.

**Comment by maikol-solis** [comment]
Yes. You certainly have a better judgement about this than I do, and my initial idea could be improved, of course. If you have a better config, I can help you to test it.

**Comment by rougier** [comment]
I'm actually using the box with nano-vertico mode (to show this is an input area). I get inspired by nyxt.

**Comment by maikol-solis** [comment]
OK. I installed nano-vertico, and it improves a lot the whole UI. However, it has a similar issue with nano-dark 

![image](https://github.com/rougier/nano-theme/assets/4078489/a4489acd-c3b4-4663-8c7e-956ba6e23041)

nano-vertico with the light theme works as expected 

![image](https://github.com/rougier/nano-theme/assets/4078489/b2d32d3d-9dd9-4a23-8dae-ccccb9ca09a6)

**Comment by rougier** [comment]
Can you open an issue on nano-vertico?

**Comment by maikol-solis** [comment]
Yes sure, no problem.

**Comment by dwla** [comment]
maikol-solis’s config of `mode-line` and `mode-line-inactive` is intuitive. I find the box (as well as the darker background) in the minibuffer a little bit noisy, it’s more comfortable for me to make `mode-line` the same as the `mode-line-inactive`. Here is my config. BTW, nano-theme works well with `doom-modeline`!!!

```
(use-package nano-theme
  :custom-face
  (mode-line-buffer-id ((t (:foreground "black" :weight bold))))
  (mode-line-emphasis ((t (:foreground "black" :weight bold))))
  (doom-modeline-buffer-modified ((t (:foreground "black" :weight bold))))
  ;; mode-line coincides with mode-line-inactive
  (mode-line ((t (:background "#90A4AE" :foreground "#FFFFFF" :box (:line-width (3 . 3) :color "#90A4AE")))))
  (nano-popout ((t (:foreground "#B22222" :weight bold))))
  (nano-salient ((t (:foreground "#1E90FF" :weight semibold))))
  (vertical-border ((t (:foreground "black"))))
  ;; config for ess
  (doom-modeline-urgent ((t (:foreground "black" :weight bold))))
  (doom-modeline-warning ((t (:foreground "black" :weight bold))))
  (doom-modeline-info ((t (:foreground "black" :weight bold))))
  ;; config for eshell
  (eshell-prompt ((t (:foreground "#B03060"))))
  (dired-flagged ((t (:foreground "#FF3030" :weight bold))))
  :config
  (load-theme 'nano-light t))
```
